save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/CloneRepositoryController.kt

```diff
@@ -132,9 +135,22 @@ class CloneRepositoryController(
         .retrieve()
         .toEntity<String>()
 
-    private fun Flux<FileInfo>.collectToMultipart(multipartBodyBuilder: MultipartBodyBuilder) = map {
-        multipartBodyBuilder.part("file", fileSystemRepository.getFile(it))
+    @Suppress("TYPE_ALIAS")
+    private fun Flux<FileInfo>.collectToMultipartAndUpdateExecution(
+        multipartBodyBuilder: MultipartBodyBuilder,
+        execution: Execution
+    ): Mono<List<MultipartBodyBuilder.PartBuilder>> {
+        val additionalFiles = StringBuilder("")
+        return map {
+            val path = Paths.get(it.uploadedMillis.toString()).resolve(it.name)
```

Comment:
You should use logic from `fileSystemRepository` to calculate the path

Reply:
Used `getPath` method from `fileSystemRepository`